app/src/main/java/com/infomaniak/swisstransfer/ui/screen/newtransfer/upload/UploadSuccessEmailScreen.kt

```diff
@@ -54,7 +54,6 @@ fun UploadSuccessEmailScreen(
     )
 }
 
-@OptIn(ExperimentalLayoutApi::class)
 @Composable
 private fun Content(emails: List<String>) {
```

Comment:
Now that you have extracted the component, remove this function content and just write it in the lambda parameter of the screen